web/src/components/pages/contribution/speak/speak.tsx

```diff
@@ -46,7 +46,7 @@ import { SentenceRecording } from './sentence-recording';
 
 import './speak.css';
 
-const MIN_RECORDING_MS = 1000;
+const MIN_RECORDING_MS = 750;
```

Comment:
Really nice catch on this! I'd be comfortable lowering it even further (500ms?) since we also have a `MIN_VOLUME` check in place.